Day-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Card.css';
+
+const CardContainer = () => {
+    return (
+      <div className = "cardContainer">
+        <a href = "reactjs.org">
+            <img src = "https://ibin.co/3wnC6SgIOJud.png" alt = "" className = "cardContainerImg"/>
+            <p>Get started with React</p>
+            <p>React makes it painless to create interactive UIs. Design simple views for each state in your application.</p>
+            <p>reactjs.org</p>
+        </a>
+      </div>
+    );
+  };
+
+export default CardContainer;
```

Comment:
If you have any questions for abstracting these lines out to their respective components, let me know.